src/LoginServer/Network/LoginPacketHandler.cs

```diff
@@ -330,7 +334,10 @@ public void CB_COMMANDER_MOVE(LoginConnection conn, Packet packet)
 			}
 
 			// Move
-			character.BarrackPosition = new Position(x, y, z);
+			// This needs to be validated such that a commander may not move to an invalid coordinate.
```

Comment:
For TODO type comments it's useful to add a `TODO:` prefix, so IDEs can pick them up and make them searchable.